migrations/Version20201012163616.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20201012163616 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE 
+          thematic_community 
+        ADD 
+          canonical_name VARCHAR(255) NOT NULL, 
+        ADD 
+          slug VARCHAR(255) NOT NULL');
```

Comment:
do we need to fill these fields for existing thematics?

Reply:
yes but, no communities exists yet